docs/book/supported_features_matrix.md

```diff
@@ -2,19 +2,28 @@
-| Multi-master for CSI driver | Yes                                                                      | Yes `*`        | Yes `*`               | Yes `*`                                  |
-| Datastores supported        | vmfs/nfs/vSAN/(vVOL `*`)                                                 | vSAN `*`       | vmfs/nfs/vSAN `*`     | vmfs/nfs/vSAN `*`                        |
-| Static Provisioning         | Yes                                                                      | Yes `*`        | No                    | Yes                                      |
-| Access mode                 | RWO                                                                      | ROX,RWX `*`    | RWO*                  | RWO `*`                                  |
-| Volume Topology/Zones       | Yes (beta)                                                               | No             | No                    | No                                       |
-| Extend Volume               | Yes (k8s 1.16+)* <br>beta                                                   | No             | No                    | No                                       |
-| Encryption                  | Yes (SPBM policy driven)*                                                | No             | No                    | No                                       |
-| K8s versions supported      | 1.14, 1.15 for CSI 1.0 features<br>1.15, 1.16, 1.17 for CSI 2.0 features | 1.16.x, 1.17.x | 1.16.x                | 1.16.x                                   |
+* If the CSI version `2.0` driver is installed on K8s running on vSphere 6.7U3, the older CSI `1.0` driver features continue to work but the new CSI `2.0` features are not supported.
+* If the CSI version `1.0.2` is installed on K8s running on vSphere 7.0, the CSI `1.0` driver features continue to work. CSI version `1.0.1` is not compatible with vSphere 7.
+* CSI version `1.0.x` and CSI version `2.0` on vSphere 6.7U3, vSphere with K8s, TKG ‘Guest’ and PKS 1.7 only support dynamically provisioned block (`RWO`) volumes on vSphere storage.
+* The dynamic creation of read-write-many (`RWX`) and read-only-many (`ROX`) file based Persistent Volumes is only available in vSphere 7.0 through vSAN 7.0 File Services.
+* PV Encryption, offline volume grow and Topology/AZ support are only available on block based Persistent Volumes. These features are not available with PVs backed by file shares.
+* Offline volume grow requires a minimum Kubernetes version of 1.16+, and is also a beta feature in this release.
+* Multi-node Control Planes only supports having a single CSI Pod being active at any time (replica = 1 in controller manifest).
+* [`WaitForFirstConsumer`](https://kubernetes.io/docs/concepts/storage/storage-classes/) is a K8s feature that delays volume binding until the Pod has been scheduled, and is used for Pod and PV placement.
+* Enterprise PKS/TGKI (v1.7) does not support vSphere 7.0 at the time of writing.
```

Comment:
@mylesagray I found following detail from https://www.vmware.com/resources/compatibility/sim/interop_matrix.php#interop&644=&2=

Can we get this statement updated based on this? 1.8 is already out and supports vSphere 7.0

<img width="1025" alt="Screen Shot 2020-07-08 at 10 24 03 AM" src="https://user-images.githubusercontent.com/22985595/86950658-47186600-c105-11ea-89dd-f12e66aeb279.png">

Another question - Do TGKI 1.8/1.7 support CSI Driver?

Reply:
@divyenpatel yes, the CSI Driver can be installed - it isn't installed by default, but it is supported.

Reply:
I removed the dedicated TKGI column, and added a note on what "native" means - as all distributions with vanilla bits fall under "native" - so that simplifies the matrix some.